.analysis_options

```diff
@@ -0,0 +1,56 @@
+#  Supported lint rules and documentation:
+#   http://dart-lang.github.io/linter/lints/
+
+analyzer:
+  strong-mode: true
+  exclude:
+    - lib/client/piratesapi.dart
+
+linter:
+  rules:
+    - avoid_empty_else
+    - cancel_subscriptions
+    - close_sinks
+    - comment_references
+    - control_flow_in_finally
+    - empty_statements
+    - hash_and_equals
+    - iterable_contains_unrelated_type
+    - list_remove_unrelated_type
+    - test_types_in_equals
+    - throw_in_finally
+    - unrelated_type_equality_checks
+    - valid_regexps
+
+#    - always_declare_return_types -- doesn't interact well with strong mode
+#    - always_specify_types        -- doesn't interact well with strong mode
+    - annotate_overrides
+#    - avoid_as                    -- jmesserly recommends avoiding
+    - avoid_init_to_null
+    - avoid_return_types_on_setters
+    - await_only_futures
+    - camel_case_types
+    - constant_identifier_names
+    - empty_catches
+    - empty_constructor_bodies
+    - implementation_imports
+    - library_names
+    - library_prefixes
+    - non_constant_identifier_names
+    - one_member_abstracts
+    - only_throw_errors
+    - overridden_fields
+    - package_api_docs
+    - package_prefixed_library_names
+    - prefer_is_not_empty
+#   - public_member_api_docs
```

Comment:
Add a comment for this? Maybe:

-- this is example code, not public APIs

(if indeed that's why we're omitting this flag)